README.md

```diff
@@ -93,24 +93,27 @@ twilio-ruby uses [Faraday](https://github.com/lostisland/faraday) to make HTTP r
 If you just need to generate a Capability Token for use with Twilio Client, you
 can do this:
 
-```ruby
+``` ruby
```

Comment:
We don't have spaces before the language in other libraries, so we should keep it consistent.